node_modules/gh-groups/lib/rest.js

```diff
@@ -23,6 +23,32 @@ var GrassHopper = require('gh-core');
 var GroupsAPI = require('./api');
 
 /**
+ * @REST getUserMemberships
+ *
+ * Get the groups a user is a member of
+ *
+ * @Server      admin,app
+ * @Method      GET
+ * @Path        /users/{id}/memberships
+ * @PathParam   {number}            id                      The id of the user to get the upcoming events for
+ * @QueryParam  {number}            [limit]                 The maximum number of results to retrieve. Default: 25
+ * @QueryParam  {number}            [offset]                The paging number of the results to retrieve
+ * @Return      {EventList}                                 The upcoming events for the user
+ */
+var getUserMemberships = function(req, res) {
```

Comment:
I would actually also prefer if we cleaned this up in one go. We don't have too many `rest.js` files, so it wouldn't take too long to go through then. If we only fix whenever we touch, we'll just be stuck with a combination of 2 approaches (in the best case for a long time).